src/Enhavo/Bundle/DemoBundle/Resources/config/app/config.yaml

```diff
@@ -40,8 +40,8 @@ enhavo_theme:
         enable: true
 
 enhavo_translation:
-    enable: false
+    enable: true
```

Comment:
Should be keep turned off until the TranslationBundle is fully tested

Reply:
reverted